docs/using.rst

```diff
@@ -188,6 +190,24 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant 2 way folder sync
+=========================
+For basic usage please refer `Vagrant documentation. <https://www.virtualbox.org/manual/ch04.html#sharedfolders>`_
+
+In some scenarios 2 way bi-directional sync makes more sense as it is the simplest way to move files (e.g. code) between 
```

Comment:
That's the default mode and the one which makes always sense. rsync is the exception. You are turning things up side down